internal/store/store.go

```diff
@@ -15,21 +15,26 @@ import (
 	"github.com/dicedb/dice/config"
 )
 
-// WatchEvent represents a change in a watched key.
-type WatchEvent struct {
+// QueryWatchEvent represents a change in a watched key.
+type QueryWatchEvent struct {
 	Key       string
 	Operation string
 	Value     object.Obj
 }
 
+type CommandWatchEvent[T any] struct {
```

Comment:
Where is this getting used?

Reply:
messed up and included this from another change I was working on, removing it.